modules/ROOT/pages/branded_desktop_client/branded_linux_desktop_clients.adoc

```diff
@@ -0,0 +1,121 @@
+= Deploy And Update Branded Linux Desktop Clients
+
+== Introduction
+
+As an ownBrander user, you can enable the build of branded Linux in the “Desktop” section of their account. You can download a `*.tar` file which contains everything to set up complete self-hosted Linux repositories
```

Comment:
```suggestion
As an ownBrander user, you can enable the build of branded Linux in the “Desktop” section of your account. You can download a `*.tar` file which contains everything to set up complete self-hosted Linux repositories
```

I am not so familiar with this stuff. Not sure who is "their" in "their account".